src/blueprints/tv2/tv2-logger.ts

```diff
@@ -0,0 +1,15 @@
+import { Level as LogLevel } from '@tv2media/logger'
+
+export { Level as LogLevel } from '@tv2media/logger'
+
+export interface Tv2Logger {
+  error(message: string, metadata?: object): void
+  warn(message: string, metadata?: object): void
+  info(message: string, metadata?: object): void
+  debug(message: string, metadata?: object): void
+  trace(message: string, metadata?: object): void
+  metadata(metadata: object): Tv2Logger
+  tag(tag: string): Tv2Logger
+  data(data: unknown): Tv2Logger
+  setLevel(level: LogLevel): void
+}
```

Comment:
This interface is already created in the logging library that we are using. There is no need to recreate it.
If the interface of the library change, we won't catch it except for a runtime error.

Just use the interface:
```
import { Logger as MediaLogger } from '@tv2media/logger'

export interface Tv2Logger extends MediaLogger {}
```

Reply:
The message of the method on this interface have the type string, whereas the interface from the library have the type unknown for messages.

Also there is no interface in the library called `MediaLogger`.

Reply:
> An advantage of defining the interface ourselves, is that if we change the logger library or the logger library API changes drastically, we only need to change the implementation of the logger instead of updating all the log statements throughout the codebase.

We wouldn't lose that advantage.
I'm not going to die on this hill, so if this is what the majority is for then lets go with it.